web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -1959,10 +1961,13 @@ private VoidResponse close(String id, Principal principal) {
                     RunningQuery query = getQueryById(id, principal);
                     close(query);
                 } catch (Exception e) {
-                    // if this is a query that is in the old query cache, then we have already successfully closed this query so ignore
-                    if (oldQueryCache.getIfPresent(id) == null) {
+                    log.warn("Failed to close " + id + ", checking if closed previously");
```

Comment:
It seems this is a case that we're expecting to happen, so we probably want to log at info or debug.

Reply:
that is reasonable, I will change the pattern